src/hotspot/share/gc/shenandoah/c2/shenandoahBarrierSetC2.cpp

```diff
@@ -1063,12 +1063,12 @@ Node* ShenandoahBarrierSetC2::ideal_node(PhaseGVN* phase, Node* n, bool can_resh
     // If one input is NULL, then step over the barriers (except LRB native) on the other input
```

Comment:
Should be `weak LRB`, not `LRB native` now? Probably text search for `native` elsewhere?